index.js

```diff
@@ -0,0 +1,11 @@
+// const { min } = require("lodash");
```

Comment:
- [ ] Kindly remove commented pieces of code to make the project more professional and cleaner 💯